src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogsExporter.cs

```diff
@@ -0,0 +1,95 @@
+// <copyright file="OtlpLogsExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NET461 || NETSTANDARD2_0 || NETSTANDARD2_1
+using System;
+using Grpc.Core;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+using OpenTelemetry.Logs;
+using OtlpCollector = Opentelemetry.Proto.Collector.Logs.V1;
+
+namespace OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// Exporter consuming <see cref="LogRecord"/> and exporting the data using
+    /// the OpenTelemetry protocol (OTLP).
+    /// </summary>
+    internal class OtlpLogsExporter : BaseOtlpExporter<LogRecord>
```

Comment:
Regarding the name, I suggest `OtlpLogExporter`:
* We have `MetricExporter` and `MetricReader`.
* The namespace (e.g. `OpenTelemetry.Logs`, `OpenTelemetry.Metrics`, `OpenTelemetry.Resources`) can be plural form, the types are normally singular form.

Reply:
+1 of using `OtlpLogExporter`

Reply:
Ah yes, I waffled on this a bit. I had intentionally pluralized the name, contrary to our regular pattern, because things are pluralized in the classes generated by the protos - classes like `ExportLogsServiceRequest` and `LogsServiceClient`. Though I do not have a strong opinion here. I'll un-pluralize.

When I landed the OTLP metric exporter, I made the same pluralization decision. So, if this is the path we want, I'll fix that in a separate PR.